operations/images.md

```diff
@@ -10,64 +11,63 @@ Returns URLs of the specified images.
 
 ```javascript
 {
-    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
-    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
-    "Client": "Sample Client 1.0.0",
-    "Images":[
-        {
-            "ImageId": "57a971a5-a335-48f4-8cd1-595245d1a876",
-            "Width": 200,
-            "Height": 150,
-            "ResizeMode": "Fit"
-        }
-    ]
+  "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+  "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+  "Client": "Sample Client 1.0.0",
+  "Images": [
+    {
+      "ImageId": "57a971a5-a335-48f4-8cd1-595245d1a876",
+      "Width": 200,
+      "Height": 150,
+      "ResizeMode": "Fit"
+    }
+  ]
 }
 ```
 
 | Property | Type | Contract | Description |
-| :--- | :--- | :--- | :--- |
-| `ClientToken` | string | required | Token identifying the client application. |
-| `AccessToken` | string | required | Access token of the client application. |
+| :-- | :-- | :-- | :-- |
 | `Client` | string | required | Name and version of the client application. |
-| `Images` | array of [Image parameters](#image-parameters) | required | Parameters of images whose URLs should be returned. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `Images` | array of [Image size parameters](#image-size-parameters) | required | Parameters of images whose URLs should be returned. |
 
-#### Image parameters
+#### Image size parameters
 
 | Property | Type | Contract | Description |
-| :--- | :--- | :--- | :--- |
+| :-- | :-- | :-- | :-- |
 | `ImageId` | string | required | Unique identifier of the image. |
-| `Width` | number | required | Desired width of the image. |
-| `Height` | number | required | Desired height of the image. |
-| `ResizeMode` | string [Image resize mode](#image-resize-mode) | required | Mode how the image should be resized to the desired width and height. |
+| `Width` | integer | optional | Desired width of the image. |
+| `Height` | integer | optional | Desired height of the image. |
+| `ResizeMode` | [Image resize mode](#image-resize-mode) | required | Mode how the image should be resized to the desired width and height. |
```

Comment:
Just thinking out loud, and this isn't necessarily a change needed right now, but I wonder if we should use the word `string` for string enumerations, and `object` for object definitions? Or do you think this isn't necessary?

Reply:
I think it would be helpful, but agree it's not needed right now. Perhaps add it as a follow up after we do the initial round?